Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -8076,7 +8087,7 @@ void HTMLMediaElement::createMediaPlayer() WTF_IGNORES_THREAD_SAFETY_ANALYSIS
     RefPtr page = document().page();
     player->setPageIsVisible(!m_elementIsHidden);
     player->setVisibleInViewport(isVisibleInViewport());
-    player->setInFullscreenOrPictureInPicture(isFullscreen());
+    player->setInFullscreenOrPictureInPicture(isInFullscreenOrPictureInPicture());
```

Comment:
This seems OK for now, but have you considered making external playback be a new kind of `VideoFullscreenMode` alongside `VideoFullscreenModeStandard`, `VideoFullscreenModePictureInPicture`, etc.?